docs/detections/prebuilt-rules-management.asciidoc

```diff
@@ -30,7 +30,11 @@ Follow these guidelines to start using the {security-app}'s <<prebuilt-rules, pr
 [role="screenshot"]
 image::images/prebuilt-rules-add-badge.png[The Add Elastic Rules page]
 
-. Click *Add Elastic rules*, then do one of the following:
+. Click *Add Elastic rules*.
++
+TIP: To view the details of a rule before you install it, select the rule name. This opens the rule details flyout.
```

Comment:
Minor terminology suggestion here because we try to avoid [ableist](https://docs.elastic.dev/content-design/inclusivity#avoid-violent-offensive-ableist-terminology) lang: 

```suggestion
TIP: To examine the details of a rule before you install it, select the rule name. This opens the rule details flyout.
```